2-ui/1-document/03-dom-navigation/article.md

```diff
@@ -22,7 +22,7 @@ Let's discuss them in more detail.
 The topmost tree nodes are available directly as `document` properties:
 
 `<html>` = `document.documentElement`
-: The topmost document node is `document.documentElement`. That's DOM node of `<html>` tag.
+: The topmost document node is `document.documentElement`. That's the DOM node of the`<html>` tag.
```

Comment:
add a space after the second "the", please.

Reply:
done